src/runway/blueprints/staticsite/staticsite.py

```diff
@@ -168,16 +360,34 @@ def create_template(self):
                 )
             )
         )
-        template.add_output(Output(
+        self.template.add_output(Output(
             'BucketName',
             Description='Name of website bucket',
             Value=bucket.ref()
         ))
+        self.template.add_output(Output(
+            'BucketWebsiteURL',
+            Condition="CFDisabled",
+            Description='URL of the bucket website',
+            Value=GetAtt(bucket, 'WebsiteURL')
```

Comment:
Please change this to `bucket.get_att('WebsiteURL')`

(doesn't change the generated template at all, really just a style/consistency thing)

Reply:
Updated